docs/reference/modules/cluster/allocation_awareness.asciidoc

```diff
@@ -17,9 +17,12 @@ The allocation awareness settings can be configured in
 `elasticsearch.yml` and updated dynamically with the
 <<cluster-update-settings,cluster-update-settings>> API.
 
-{es} prefers using shards in the same location (with the same
-awareness attribute values) to process search or GET requests. Using local
-shards is usually faster than crossing rack or zone boundaries.
+By default {es} uses <<search-adaptive-replica,adaptive replica selection>>
+to route search or GET requests. However, with the presence of allocation awareness
+attributes {es} will prefer using shards in the same location (with the same
+awareness attribute values) to process these requests. This behavior can be
+disabled by specifying `export ES_JAVA_OPTS="-Des.search.ignore_awareness_attributes=true"`
```

Comment:
This recommendation might override existing `ES_JAVA_OPTS`, should we do something like below?

```suggestion
disabled by specifying `export ES_JAVA_OPTS="$ES_JAVA_OPTS -Des.search.ignore_awareness_attributes=true"`
```